File: Programmars/Level_1/문자열 내 마음대로 정렬하기.py
def solution(strings, n):
    return sorted(strings, key=lambda x: (x[n], x[:n], x[n+1:]))


print(solution(["sun", "bed", "car"], 1))  # ["car", "bed", "sun"]
print(solution(["abce", "abcd", "cdx"], 2))  # ["abcd", "abce", "cdx"]

File: JavaScript/Programmars/Level_1/문자열 내 마음대로 정렬하기.js
function solution(strings, n) {
  return strings.sort().sort((a, b) => a[n].charCodeAt() - b[n].charCodeAt());
}

console.log(solution(['sun', 'bed', 'car'], 1)); // [ 'car', 'bed', 'sun' ]
console.log(solution(['abce', 'abcd', 'cdx'], 2)); // [ 'abcd', 'abce', 'cdx' ]
